packages/react-components/react-migration-v8-v9/library/project.json

```diff
@@ -4,5 +4,15 @@
   "projectType": "library",
   "sourceRoot": "packages/react-components/react-migration-v8-v9/library/src",
   "tags": ["vNext", "v8", "platform:web"],
-  "implicitDependencies": []
+  "implicitDependencies": [],
+  "targets": {
+    "test": {
+      "dependsOn": [
+        {
+          "target": "build",
+          "projects": ["react", "react-hooks", "fluent2-theme"]
```

Comment:
we need only non v9 deps to build instead of all v9 + v8 == significantly faster execution